### PR TITLE
3 random "incompetence" messages

### PR DESCRIPTION
**Pull request by MrKickkiller**
With ability to get more or less, by changing 1 variable (in EventHandler.java) and adding the according amount or more of unique incompetence messages in en_US.lang

### CODE REVIEW

File: out/production/Eureka/assets/eureka/lang/en_US.lang [hunk 1]
```diff
@@ -0,0 +1,39 @@
+item.engineeringDiary.name=Engineering Diary
```
Comment (AEnterprise) [inline out/production/Eureka/assets/eureka/lang/en_US.lang]: please remove the additional file

File: src/main/resources/assets/eureka/lang/en_US.lang [hunk 2]
```diff
@@ -11,7 +11,7 @@ eureka.dusterTier2-2Finished=You like that you don't have to jump on the machine
 eureka.kineticToolFinished=While making yet another tool to replace your old one you think of a way to make a tool that won't break
 eureka.missingKnowledge=You have no idea how this machine works (yet). You try to use it, but due to your incompetence, it falls apart.
 eureka.missingKnowledgeKineticTool=You have no idea how this tool works (yet). You try to use it, but due to your incompetence, it falls apart.
-
+//TODO: Add X more callouts for this situation ^^
```
Comment (AEnterprise) [inline src/main/resources/assets/eureka/lang/en_US.lang]: remove the TODO and put the messages in please

**Comment by AEnterprise** [comment]
added some line notes and could you first test it and then squash the commits?

File: src/main/java/eureka/core/EventHandler.java [hunk 3]
```diff
@@ -83,13 +89,15 @@ public void onCrafted(PlayerEvent.ItemCraftedEvent event) {
 			if (!key.isEmpty())
 				EurekaKnowledge.makeProgress(event.player, key, 1);
 		}
-
+//		@SubscribeEvent
+//		public void onCommand(PlayerEvent.)
```
Comment (AEnterprise) [inline src/main/java/eureka/core/EventHandler.java]: this is also unneeded

**Comment by AEnterprise** [comment]
you better do this over again, it's a bit of a mess

**Comment by MrKickkiller** [comment]
I'm still a noob :/

**Pull request closed without merging**